src/Symfony/Component/Notifier/Bridge/Firebase/composer.json

```diff
@@ -21,7 +24,9 @@
         "symfony/notifier": "^5.0,<5.2"
     },
     "autoload": {
-        "psr-4": { "Symfony\\Component\\Notifier\\Bridge\\Firebase\\": "" },
+        "psr-4": {
+            "Symfony\\Component\\Notifier\\Bridge\\Firebase\\": ""
+        },
```

Comment:
@fabpot I agree with the keywords, but shall we keep this in one line or readable like this. In the 5.2 branch we have it mixed 🤔 

If you don't care, please close the PR 👍 Thanks